fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/RdfContext.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.api;
+
+/**
+ * A collection of RDF contexts that can be used to extract triples from FedoraResources
+ *
+ * @author acoburn
+ * @since Dec 4, 2015
+ */
+public enum RdfContext {
```

Comment:
This is more what I would be hoping for. @awoods , @escowles , @barmintor can I get you to glance at this? Is this in line with our prospective moves towards developing versioning, fixity, etc. as extensions to LDP? I would think that anything that develops from a Prefer header value or as an extension based on LDP type would show up here in some reasonably explicit way.

I'm still not happy about having to expose this stuff using an `enum` instead of polymorphism, but I don't want to drag us around and around with that right now. The question I want to be a little more sure about is whether we are capturing here the "flavors" of RDF that any Fedora impl would have to be able to supply, to the best of our current ability.